languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/grammar_csv2xml.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  test.py
+#   Convert a file which contains grammar rules into grammar xml format for LanguageTool
+#   The text file contains linguistic rules from book of "Guide of Commons errors" n by  Marwan Albawab
```

Comment:
I suppose that "n by" should be "by".
Also only put one space rather than 2 after "by".